test/libsolidity/gasTests/abiv2_optimised.sol

```diff
@@ -17,9 +17,9 @@ contract C {
 // optimize-yul: true
 // ----
 // creation:
-//   codeDepositCost: 603000
-//   executionCost: 638
-//   totalCost: 603638
+//   codeDepositCost: 600400
+//   executionCost: 632
+//   totalCost: 601032
```

Comment:
https://gist.github.com/hrkrshnn/e622364b543d6e26a70266e25e88a48a

A diff of the changes.

Reply:
This looks good!